src/main/java/javaserver/Routes.java

```diff
@@ -3,21 +3,38 @@
 import java.util.HashMap;
 
 public class Routes {
+	public static final String[] FILES = {"/file1", "/text-file.txt"};
+
 	private static final String[] SUPPORTED_ROOT_REQUESTS = {"GET"};
 	private static final String[] SUPPORTED_FORM_REQUESTS = {"GET", "POST", "PUT"};
 	private static final String[] SUPPORTED_METHOD_OPTIONS = {"GET", "HEAD", "POST", "OPTIONS", "PUT"};
+	private static final String[] SUPPORTED_FILE_REQUESTS = {"GET"};
+
+	private static final String[] FOUND_REDIRECT_REQUESTS = {"GET"};
```

Comment:
I'm curious if this is a potential violation of the Open/Closed principle as every time you need to add a new route and constant of its supported HTTP methods, you need to modify the Routes class. I think this might be okay for now, but I'd like you to keep this in mind as this continues to grow.